components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -308,6 +308,8 @@ be lost if SCREAM_HACK_XML is not enabled.
       <do_aerosol_rad>true</do_aerosol_rad>
       <do_aerosol_rad COMPSET=".*SCREAM.*noAero">false</do_aerosol_rad>
       <enable_column_conservation_checks>false</enable_column_conservation_checks>
+      <extra_clnclrsky_diag>false</extra_clnclrsky_diag>
```

Comment:
Would you mind adding the `type="logical"` specifier here, and perhaps, also a `doc="blah"` string? It helps with the auto-generated documentation.

Reply:
If you don't mind, and you have time to spare (haha), you could also add it for some of the other vars here (not the ones inherited from `atm_proc_base`, only the rrtmgp-specific ones).